ecosystem/typescript/sdk/src/aptos_account.test.ts

```diff
@@ -60,7 +60,7 @@ test("Serializes/Deserializes", () => {
 
 test("Signs Strings", () => {
   const a1 = AptosAccount.fromAptosAccountObject(aptosAccountObject);
-  expect(a1.signHexString("0x77777").hex()).toBe(
+  expect(a1.signHexString("0x7777").hex()).toBe(
```

Comment:
There's a bug in the existing Aptos code which ignores incomplete bytes-- this change just fixes that test case.